src/pages/candidate-details/__tests__/candidate-details.spec.ts

```diff
@@ -37,6 +37,18 @@ describe('CandidateDetailsPage', () => {
     it('should create', () => {
       expect(component).toBeDefined();
     });
+
+    it('specialNeedsIsPopulated returns true for a populated array', () => {
```

Comment:
`specialNeedsIsPopulated` could be a `describe` which holds these 2 test cases together, so that we have a english readable sentence as a test name.

Reply:
now nested under a describe.